web/lti.go

```diff
@@ -21,15 +21,17 @@ func (w *Web) InitLti() {
 
 func loginWithLTI(c *Context, w http.ResponseWriter, r *http.Request) {
 	mlog.Debug("Received an LTI Login request")
-
-	LTISettings, ltiErr := c.App.GetLTISettings();
-	if(ltiErr != nil){
-		mlog.Error(ltiErr.Error());
+	//
+	// Get the LTI Settings from plugin config.
+	//
```

Comment:
this comment is not helpful. it's better if you remove this and similar comments.